src/includes/getting-started-config/node.mdx

```diff
@@ -1,9 +1,29 @@
-You need to inform the Sentry Node SDK about your DSN:
+Once this is done, Sentry's Node SDK captures all unhandled exceptions and transactions.
```

Comment:
It kinda sounds like transactions are "unhandled" as well 🤔  Maybe switching the order to `transactions and unhandled exceptions` would make it more explicit?

Reply:
Also, code samples below won't capture any transactions. It requires either manual setup + HTTP integration, or additional all-in-one integration, like the one we have for Express.

Reply:
I've switched xns and unhandled exceptions. 

@HazAT - can I get some help with a correct code sample that covers capturing xns?